third_party/docfx/templates/devsite/token.json

```diff
@@ -22,20 +22,20 @@
   "improveThisDoc": "Improve this Doc",
   "viewSource": "View Source",
   "inheritance": "Inheritance",
-  "inheritedMembers": "Inherited Members",
-  "derived": "Derived Types",
+  "inheritedMember": "Inherited Member",
+  "derived": "Derived Type",

```

Comment:
I don't think these changes are always correct. There are some classes that have multiple derived types (same with the other changes).

Reply:
I've added snippets to make them into plural form when necessary, for example:

```<h2>{{__global.inheritedMember}}{{#inheritedMembers.1}}s{{/inheritedMembers.1}}</h2>```

will that make it okay? Or would you rather keep them simply only in plural form?

Reply:
I'm really just not sure about this. I'm fine to change it if that's what's best. But, it seems like having it plural is OK? "Here is the list of derived types: [single derived type]" -- that seems OK to me.

@jennyrosewood, thoughts?

Reply:
Jenny and I had a chat offline to discuss this and thus expanded this topic from just examples to anything else that applied. Sorry, should've noted that this was based on discussions I had with Jenny.